sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,630 @@
+
+    for selection_mark in content.selection_marks:
+        print("Selection mark is '{}' within bounding box '{}' and has a confidence of {}".format(
+            selection_mark.state,
+            format_bounding_box(selection_mark.bounding_box),
+            selection_mark.confidence
+        ))
+    print("----------------------------------------")
+```
+
+
+Analyzing document layout with `3.2.x`:
+```python
+with open(path_to_sample_documents, "rb") as f:
+    poller = document_analysis_client.begin_analyze_document(
+        "prebuilt-layout", document=f
+    )
+result = poller.result()
+
+for idx, style in enumerate(result.styles):
+    print(
+        "Document contains {} content".format(
+            "handwritten" if style.is_handwritten else "no handwritten"
+        )
+    )
+
+for idx, page in enumerate(result.pages):
+    print("----Analyzing layout from page #{}----".format(idx + 1))
+    print(
+        "Page has width: {} and height: {}, measured with unit: {}".format(
+            page.width, page.height, page.unit
+        )
+    )
+
+    for line_idx, line in enumerate(page.lines):
+        print(
+            "Line # {} has text content '{}' within bounding box '{}'".format(
+                line_idx,
+                line.content,
+                format_bounding_box(line.bounding_box),
+            )
+        )
+
+    for word in page.words:
+        print(
+            "...Word '{}' has a confidence of {}".format(
+                word.content, word.confidence
+            )
+        )
+
+    for selection_mark in page.selection_marks:
+        print(
+            "Selection mark is '{}' within bounding box '{}' and has a confidence of {}".format(
+                selection_mark.state,
+                format_bounding_box(selection_mark.bounding_box),
+                selection_mark.confidence,
+            )
+        )
+
+for table_idx, table in enumerate(result.tables):
+    print(
+        "Table # {} has {} rows and {} columns".format(
+            table_idx, table.row_count, table.column_count
+        )
+    )
+    for region in table.bounding_regions:
+        print(
+            "Table # {} location on page: {} is {}".format(
+                table_idx,
+                region.page_number,
+                format_bounding_box(region.bounding_box),
+            )
+        )
+    for cell in table.cells:
+        print(
+            "...Cell[{}][{}] has text '{}'".format(
+                cell.row_index,
+                cell.column_index,
+                cell.content,
+            )
+        )
+        for region in cell.bounding_regions:
+            print(
+                "...content on page {} is within bounding box '{}'".format(
+                    region.page_number,
+                    format_bounding_box(region.bounding_box),
+                )
+            )
+
+print("----------------------------------------")
+```
+
+Analyzing general prebuilt document types with `3.2.x`:
```

Comment:
I feel like we generally go 3.1.x and then show 3.2.x but this example doesn't have its 3.1.x counterpart. I know prebuilt-document didn't exist in 3.1.x, but I think we should include a sentence about how this should replace training without labels.